common/source/docs/common-airvolute-DroneCore-Suite.rst

```diff
@@ -0,0 +1,134 @@
+.. _common-airvolute-DroneCore-Suite:
```

Comment:
we will need to add this in common-autopilots.rst  as a TOC entry....

Reply:
OK, I'd prefer to have the page ready and then to add it to common-autopilots